src/main/java/me/qoomon/gradle/gitversioning/GitVersioningPluginExtension.java

```diff
@@ -673,7 +686,11 @@ private static String slugify(String value) {
     }
 
     private static String increaseStringNumber(String majorVersion) {
-        return String.format("%0" + majorVersion.length() + "d", Long.parseLong(majorVersion) + 1);
+        return increaseStringNumberBy(majorVersion, 1);
+    }
+
+    private static String increaseStringNumberBy(String toIncrease, long by) {
```

Comment:
please rename to `increase(String number, long increment)`

Reply:
Done - see latest commit